contracts/upgradeable_contracts/factories/ForeignBridgeFactory.sol

```diff
@@ -0,0 +1,189 @@
+pragma solidity 0.4.24;
+
+import "../../IBridgeValidators.sol";
+import "../../IForeignBridge.sol";
+import "../../upgradeability/EternalStorageProxy.sol";
+import "../../upgradeability/EternalStorage.sol";
+import "../EternalOwnable.sol";
+
+contract ForeignBridgeFactory is EternalStorage, EternalOwnable {
```

Comment:
I think it could be a good idea to include some reference to the type of bridge the factory is about on the name of the contracts. Some options could be `HomeErcToErcBridgeFactory`, `ForeignErcToErcBridgeFactory` and `ErcToErcBridgeMapper`. Does it makes sense @akolotov ?

Reply:
As I don't really see a real value for a native (from\to) erc factory I am not sure if this change is a must have one.